events.html

```diff
@@ -145,7 +145,7 @@
         />
       </a>
       <section class="event-details">
-        <h2 class="event-upcoming">Upcoming Events</h2>
+        <h2 class="event-upcoming">Recap of our past events</h2>
```

Comment:
I think only `Past Events` will be better here, what do you think on this ?

Reply:
Sure, changed.